State_manager_code/StateManager.py

```diff
@@ -1,7 +1,8 @@
 import time
 from enum import Enum
 
-class States (Enum):
+class States(Enum):
+
```

Comment:
This is an old version of the state manager. I prefer my version. Let's talk about it.

Reply:
changed